addons/sale_margin/models/sale_order.py

```diff
@@ -10,7 +10,7 @@ class SaleOrderLine(models.Model):
         "Margin", compute='_compute_margin',
         digits='Product Price', store=True, groups="base.group_user")
     margin_percent = fields.Float(
-        "Margin (%)", compute='_compute_margin', store=True, groups="base.group_user")
+        "Margin (%)", compute='_compute_margin', store=True, groups="base.group_user", group_operator="avg")
```

Comment:
can you remove the avg ?

Reply:
It's kept for the specific case where users add the `margin_percent` in a SO list view.

Reply:
but if you group, the value is wrong (it is better to show 13400% than a 70% wrong value)

Reply:
what do you think about my fix ?

```python
    @api.model
    def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=False, lazy=True):
        if 'margin_percent' in fields:
              fields = list(set( fields + ['amount_untaxed', 'margin']))
        res = super().read_group(domain, fields, groupby, offset=offset, limit=limit, orderby=orderby, lazy=lazy)
        for r in res:
            if r['amount_untaxed']:
                r['margin_percent'] = 100 * r['margin'] / r['amount_untaxed']
            else:
                r.pop('margin_percent', None)
        return res
```

Reply:
> the point is not to have an exact computation in all cases, the point is to have at least one computation that makes sense
> e.g. you can say "on average, for customer X my margin is X%" (when grouping by partner, for ex.)
> 
> overriding the read_group seems like overkill for this (imho)